### PR TITLE
docs: clarify queries' usage

### PR DESCRIPTION
**Pull request by gcroci2**

### CODE REVIEW

**Review by DaniBodor — COMMENTED** [review]
Nice.
The other thing I felt was missing from the README is what we can actually do with it. In other words, what are `ProteinProteinInterfaceQuery` and `SingleResidueVariantQuery`?

In the very top of the README, we just state "DeepRank2 is an open-source deep learning (DL) framework for data mining of protein-protein interfaces (PPIs) or single-residue variants (SRVs)."
I think this statement warrants a little bit of extra information. Shall we do this in the same PR, or make a separate one?

File: README.md [hunk 1]
```diff
@@ -165,13 +165,21 @@ For more details, see the [extended documentation](https://deeprank2.rtfd.io/).
...
 - optionally, the correspondent position-specific scoring matrices (PSSMs), in the form of `.pssm` files.
 
+Then in particular, for the `SingleResidueVariantQuery` child class:
+
+- `chain_ids` represents the chain identifier of the variant residue (generally a single capital letter). Note that this does not limit the structure to residues from this chain. The structure contained in the `.pdb` can thus have any number of chains.
+
+For the `ProteinProteinInterfaceQuery` child class:
+
+- `chain_ids` represents the chain identifiers of the interacting interfaces (generally a single capital letter each). Note that this does not limit the structure to residues from these chains. But `chain_ids` must contain exactly 2 chains, since right now the code-base handles mono interfaces only.
+
```
[the remark this answers is not in the document]
Comment (gcroci2) [reply README.md]: I like it more :) I slightly edited and added it.

**Comment by gcroci2** [comment]
> Nice. The other thing I felt was missing from the README is what we can actually do with it. In other words, what are `ProteinProteinInterfaceQuery` and `SingleResidueVariantQuery`?
> 
> In the very top of the README, we just state "DeepRank2 is an open-source deep learning (DL) framework for data mining of protein-protein interfaces (PPIs) or single-residue variants (SRVs)." I think this statement warrants a little bit of extra information. Shall we do this in the same PR, or make a separate one?

I've done it here, give it a look :)

File: README.md [hunk 2]
```diff
@@ -17,7 +17,12 @@
...
 
-DeepRank2 allows for transformation of (pdb formatted) molecular data into 3D representations (either grids or graphs) containing structural and physico-chemical information, which can be used for training neural networks. DeepRank2 also offers a pre-implemented training pipeline, using either [CNNs](https://en.wikipedia.org/wiki/Convolutional_neural_network) (for grids) or [GNNs](https://en.wikipedia.org/wiki/Graph_neural_network) (for graphs), as well as output exporters for evaluating performances.
+As input, DeepRank2 takes [PDB-formatted](https://www.cgl.ucsf.edu/chimera/docs/UsersGuide/tutorials/pdbintro.html) atomic structures, and map them to graphs, where nodes can represent either residues or atoms, as chosen by the user, and edges represent the interactions between them. The user can configure two types of 3D structures as input for the featurization phase:
```
Comment (DaniBodor) [inline README.md:20]: The structures are (or at least can be) the same between the two, it's the problem you're trying to solve with them that is different in the two query types.

```suggestion
As input, DeepRank2 takes [PDB-formatted](https://www.cgl.ucsf.edu/chimera/docs/UsersGuide/tutorials/pdbintro.html) atomic structures, and map them to graphs, where nodes can represent either residues or atoms, as chosen by the user, and edges represent the interactions between them. DeepRank2 has the option to choose between two types of queries as input for the featurization phase:
```

**Review by DaniBodor — APPROVED** [review]
Really good improvements!
I left one final suggestion to the PPI/SRV explanation.

I only looked at the README, I assume that the other two documents are copies from here, right?

File: README.md (hunk 2 above)
Comment (gcroci2) [reply README.md]: Fair point! Edited :)

**Pull request merged**